package.json

```diff
@@ -1,12 +1,17 @@
 {
   "name": "eslint-config-discourse",
-  "version": "1.1.2",
+  "version": "1.1.3",
   "description": "Shareable eslint config for Discourse and plugins",
   "main": "index.js",
   "dependencies": {
-    "babel-eslint": "^8.2",
-    "eslint": "^4.19",
-    "prettier": "^1.19.1"
+    "ember-template-lint": "^2.11.0",
+    "eslint-plugin-lodash": "^7.1.0",
```

Comment:
Is this overkill now that lodash is deprecated in core?

Reply:
Guess we can keep it a little bit more given this is used in plugins too